test/CrossImport/skip-submodules.swift

```diff
@@ -0,0 +1,16 @@
+// Make sure we don't consider submodules, because there are...a lot of them.
+
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -enable-cross-import-overlays -Xllvm -debug-only=swift-name-binding %s 2>%t.txt
```

Comment:
In addition, I believe this test should have a `REQUIRES: asserts` as it fails for me on Android when the tests are run without assertions:
```
swift (LLVM option parsing): Unknown command line argument '-debug-only=swift-name-binding'.  Try: 'swift (LLVM option parsing) --help'
swift (LLVM option parsing): Did you mean '--debug-pass=swift-name-binding'?
```